index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+
+
+</head>
+<body>
+    <nav class="navigation">
+       <img src="./logos/MyLogo.png" alt="navigation icon" class="navigation__logo">
+       <img src="./logos/menu.svg" alt="navigation icon" class="navigation__hamburger">
+    </nav>
```

Comment:
- [ ] The toolbar (header) section should be wrapped in `<header>` tag. Then, with in it you will have the `logo` and the `nav`. The general structure would look like something like this. 
```
<header>
   <a href> Logo </a>
   <nav> </nav>
</header>
```

- [ ] You should add `logo` in the toolbar as in the design since the Figma template you have chosen (template 1) has a logo. You may also replace the `My Logo` text with your name or any other.